src/player.js

```diff
@@ -107,9 +116,14 @@ class Player extends FakeEventTarget {
   //  <editor-fold desc="Playback Interface">
   /**
    * Start/resume playback
-   * @returns {void}
+   * @returns {Promise} - The play promise
    */
-  play(): void {
+  play(): Promise {
```

Comment:
It's better not to overload the play method with our own play promise.
play is already a promise in browsers and it has annoying nature of throwing exceptions when load is called during play.
It's OK to just wait for our internal load to finish and just call play then without a promise.
Also, the interface still says play returns void...